src/ap/hostapd.h

```diff
@@ -154,6 +154,13 @@ struct hostapd_sae_commit_queue {
  * struct hostapd_data - hostapd per-BSS data structure
  */
 struct hostapd_data {
+#if defined(__ZEPHYR__) && defined(HOSTAPD)
+	/** Zephyr has coexistence of hostapd and wpa_supplicant.
+	 * To verify if ctx is hostapd or wpa_supplicant in wpa_msg,
+	 * this should be the first parameter
+	 */
+	int is_hostapd;
```

Comment:
As discussed earlier in the hostap ML, there is a clean way to do this (hostap and wpa_s in parallel, using registration of ops), I am accepting this patch on the assumption that we will start work on a proper implementation than these hacks.

Reply:
can we go with this temproraily and I'll revert it after you merged proper impl?

Reply:
Sure